sources/packages/forms/src/form-definitions/sfaa2023-24.json

```diff
@@ -16506,12 +16305,13 @@
                         "unique": false
                     },
                     "errorLabel": "",
+                    "errors": "",
                     "key": "numberOfParents",
                     "tags": [],
                     "properties": {},
                     "conditional": {
                         "show": true,
-                        "when": "isYourSpouseACanadianCitizen",
+                        "when": "doYourParentSStepParentSponsorLegalGuardianHaveAValidSinNumber",
```

Comment:
this is added by me as part of my change

Reply:
I didn't @guru-aot . what is the bug here? name of the variable?

Reply:
no the number of parents question is associated with the different yes or no, so changed it.